### ISSUE TITLE
Cucumber client is not working and it throws error as "Cucumber Language Server client: couldn't create connection to server."

### ISSUE THREAD

**Issue by BavithiranOneMindIndia** [cucumber/vscode#169]
While opening the  .Feature file it throws an error

[Error - 2:51:10 PM] Server initialization failed.
  Message: Request initialize failed with message: abort(Assertion failed: bad export type for `tree_sitter_c_sharp_external_scanner_create`: undefined). Build with -s ASSERTIONS=1 for more info.
  Code: -32603 
[Error - 2:51:10 PM] Cucumber Language Server client: couldn't create connection to server.
  Message: Request initialize failed with message: abort(Assertion failed: bad export type for `tree_sitter_c_sharp_external_scanner_create`: undefined). Build with -s ASSERTIONS=1 for more info.
  Code: -32603

**Comment by lucasbeninca** [comment]
I have the same problem
Windows 10 home.

**Comment by xeger** [comment]
Thank you for the bug report. This is fixed in [v1.8.0](https://github.com/cucumber/vscode/releases/tag/release%2Fv1.8.0) which should be published to the Visual Studio Code Marketplace soon, or you can install from the `.vsix` file in the linked release.

**Comment by alexanderilyin** [comment]
Same happens on v1.10.0:

```
[Error - 12:36:00 PM] Cucumber Language Server client: couldn't create connection to server.
  Message: Request initialize failed with message: bad export type for `tree_sitter_c_sharp_external_scanner_create`: undefined
  Code: -32603 
```

```
❯ code --version
1.94.2
384ff7382de624fb94dbaf6da11977bba1ecd427
x64
```

```
⬢ [Docker] ❯ code --list-extensions --show-versions
Extensions installed on Dev Container @ desktop-linux:
42crunch.vscode-openapi@4.29.2
a-h.templ@0.0.29
codeium.codeium@1.22.5
cucumberopen.cucumber-official@1.10.0
davidanson.vscode-markdownlint@0.56.0
dbaeumer.vscode-eslint@3.0.10
golang.go@0.42.1
mongodb.mongodb-vscode@1.9.1
ms-vscode.live-server@0.4.15
ms-vscode.makefile-tools@0.11.13
postman.postman-for-vscode@1.3.0
redhat.vscode-yaml@1.15.0
ryanluker.vscode-coverage-gutters@2.12.0
yzhang.markdown-all-in-one@3.6.2
```
